test/gtest/ucp/test_ucp_wakeup.cc

```diff
@@ -117,6 +117,13 @@ UCS_TEST_P(test_ucp_wakeup, tx_wait, "ZCOPY_THRESH=10000")
     std::string send_data(COUNT, '2'), recv_data(COUNT, '1');
     void *sreq, *rreq;
 
+    if (GetParam().transports.front().compare("tcp") == 0) {
```

Comment:
Does TCP always go front? Do we need take into account other transports here?
Pls fix commit message: "UCT/GTEST" -> "GTEST/UCP"
Minor: using `std::string::operator==` would improve readability